app/controllers/v3/service_credential_bindings_controller.rb

```diff
@@ -181,6 +179,18 @@ def enqueue_bind_job(binding_guid, message)
     pollable_job.guid
   end
 
+  def enqueue_key_job(binding_guid, message)
+    bind_job = VCAP::CloudController::V3::CreateBindingAsyncJob.new(
+      :key,
```

Comment:
`key` arg is the only difference between this and `enqueue_bind_job` maybe just extract that as a param and use `enqueue_bind_job(key, binidng_guid, message)`